src/js/components/DataTable/DateCell.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import moment from 'moment';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import TableCell from 'components/DataTable/TableCell';
+
+const DateCell = ({ tableDateFormat, ...row }) => (
+  <TableCell
+    {...row}
+    value={row.value && moment(row.value).format(tableDateFormat)}
+    defaultValue="-"
+  />
+);
+
+const mapStateToProps = state => ({
```

Comment:
is there a reason why are not using `useSelector` hook to access redux store values ?

Reply:
`mapStateToProps` and `useSelector` are replaceable, and it's just my own preference to use `mapStateToProps` inside the components and the `useSelector` for e.g. hooks. If it bothers you, I could change it, but answering the question in a nutshell - it's just a personal preference

Reply:
ok, I understand
I marked it as a comment and not a change request, just wanted to know if there is a technical reason for not using hooks
but I have nothing against the current version

Reply:
Just out of curiosity, I would like to know more about this. Can you provide a basic ELI5 for why you would use mapStateToProps over the useSelector hook?

I've read the following post on StackOverflow so just want to hear your thoughts.
https://stackoverflow.com/questions/59748180/should-i-use-useselector-usedispatch-instead-of-mapstatetoprops